### PR TITLE
Parity

### PR DESCRIPTION
**Pull request by HernanRivasAcosta**

### CODE REVIEW

File: src/kafkerl_broker_connection.erl [hunk 1]
```diff
@@ -185,20 +187,21 @@ handle_tcp_data(Bin, State = #state{connector = Connector, ets = EtsName,
                 end),
           case handle_errors(Errors, Messages, Name) of
```
Comment (elvisci) [inline src/kafkerl_broker_connection.erl]: The expression on line 188 and column 11 is nested beyond the maximum level of 3.

File: src/kafkerl_connector.erl [hunk 2]
```diff
@@ -295,41 +306,41 @@ request_metadata(Pid, Brokers, TCPOpts, Retries, RetryInterval, Request) ->
...
     {ok, Socket} ->
       % On success, send the metadata request
       case gen_tcp:send(Socket, Request) of
         {error, Reason} ->
           warn_metadata_request(Host, Port, Reason),
           % Unable to send request, try the next broker
-          request_metadata(T, TCPOpts, Request);
+          do_request_metadata(T, TCPOpts, Request);
         ok ->
           case gen_tcp:recv(Socket, 0, 6000) of
```
Comment (elvisci) [inline src/kafkerl_connector.erl]: The expression on line 331 and column 11 is nested beyond the maximum level of 3.

File: src/kafkerl_connector.erl [hunk 3]
```diff
@@ -295,41 +306,41 @@ request_metadata(Pid, Brokers, TCPOpts, Retries, RetryInterval, Request) ->
...
-      request_metadata(Pid, Brokers, TCPOpts, NewRetries, RetryInterval,
+      do_request_metadata(Pid, Brokers, TCPOpts, NewRetries, RetryInterval,
                        Request)
   end.
 
-request_metadata([], _TCPOpts, _Request) ->
+do_request_metadata([], _TCPOpts, _Request) ->
   {error, all_down};
-request_metadata([{Host, Port} = _Broker | T] = _Brokers, TCPOpts, Request) ->
+do_request_metadata([{Host, Port} = _Broker | T] = _Brokers, TCPOpts, Request) ->
```
Comment (elvisci) [inline src/kafkerl_connector.erl]: Line 315 is too long: do_request_metadata([{Host, Port} = _Broker | T] = _Brokers, TCPOpts, Request) ->.

Comment (HernanRivasAcosta) [inline src/kafkerl_connector.erl]: 1 character dude! Just one!

**Pull request merged**